### PR TITLE
Fix `create_axes_orientation_box` unused argument

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

This is the follow-up PR of #5374.
Remove `create_axes_orientation_box` unused argument.

### Details

- Comment out `GetTextEdgesProperty().SetColor` as it doesn't work.

### CODE REVIEW

File: pyvista/plotting/tools.py [hunk 1]
```diff
@@ -381,8 +370,6 @@ def create_axes_orientation_box(
         axes_actor.SetZMinusFaceText(f"-{zlabel}")
     axes_actor.SetFaceTextVisibility(not labels_off)
     axes_actor.SetTextEdgesVisibility(False)
-    # axes_actor.GetTextEdgesProperty().SetColor(edge_color.float_rgb)
-    # axes_actor.GetTextEdgesProperty().SetLineWidth(line_width)
```
Comment (banesullivan) [inline pyvista/plotting/tools.py:385]: Should we try to see if this works?

File: pyvista/plotting/tools.py [hunk 2]
```diff
@@ -380,9 +384,9 @@ def create_axes_orientation_box(
         axes_actor.SetZPlusFaceText(f"+{zlabel}")
         axes_actor.SetZMinusFaceText(f"-{zlabel}")
     axes_actor.SetFaceTextVisibility(not labels_off)
-    axes_actor.SetTextEdgesVisibility(False)
-    # axes_actor.GetTextEdgesProperty().SetColor(edge_color.float_rgb)
-    # axes_actor.GetTextEdgesProperty().SetLineWidth(line_width)
+    axes_actor.SetTextEdgesVisibility(show_text_edges)
+    axes_actor.GetTextEdgesProperty().SetColor(edge_color.float_rgb)
```
Comment (tkoyama010) [inline pyvista/plotting/tools.py:388]:
```suggestion
# https://github.com/pyvista/pyvista/pull/5382
    axes_actor.GetTextEdgesProperty().SetColor(edge_color.float_rgb)
```

File: pyvista/plotting/tools.py [hunk 3]
```diff
@@ -380,9 +384,10 @@ def create_axes_orientation_box(
         axes_actor.SetZPlusFaceText(f"+{zlabel}")
         axes_actor.SetZMinusFaceText(f"-{zlabel}")
     axes_actor.SetFaceTextVisibility(not labels_off)
-    axes_actor.SetTextEdgesVisibility(False)
-    # axes_actor.GetTextEdgesProperty().SetColor(edge_color.float_rgb)
-    # axes_actor.GetTextEdgesProperty().SetLineWidth(line_width)
+    axes_actor.SetTextEdgesVisibility(show_text_edges)
+# https://github.com/pyvista/pyvista/pull/5382
```
Comment (tkoyama010) [inline pyvista/plotting/tools.py:388]:
```suggestion
#   https://github.com/pyvista/pyvista/pull/5382
```

File: pyvista/plotting/tools.py [hunk 4]
```diff
@@ -380,9 +384,10 @@ def create_axes_orientation_box(
...
+#    axes_actor.GetTextEdgesProperty().SetColor(edge_color.float_rgb)
```
Comment (tkoyama010) [inline pyvista/plotting/tools.py:389]:
```suggestion
#   axes_actor.GetTextEdgesProperty().SetColor(edge_color.float_rgb)
```

**Comment by tkoyama010** [comment]
pre-commit.ci autofix

**Review by banesullivan — APPROVED** [review]
Nice! Thanks for going the extra mile on this one @tkoyama010

**Pull request merged**